.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+AllCops:
+  TargetRubyVersion: 2.7
```

Comment:
Do we want this to be 2.7 (the lowest we support) or 3.2 (the highest we support)?

Reply:
We'd want this to be the oldest version we support. From the [docs](https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version):

> Users may let RuboCop know the oldest version of Ruby which your project supports with:

```yaml
AllCops:
  TargetRubyVersion: 2.5
```

Reply:
We're not testing 2.5 anymore.  I'd suggest leaving to 2.7.  We should not support < 2.7.  We honestly shouldn't support 2.7, it's EOL.